test/solveEquation/solveEquation.test.js

```diff
@@ -22,8 +22,7 @@ function testSolve(equationString, outputStr, debug=false) {
 
 describe('solveEquation for =', function () {
   const tests = [
-    // can't solve this because two symbols: g and x -- so there's no steps
```

Comment:
no we actually can solve this.

Reply:
interesting, I wonder why this got fixed?

Reply:
Because of the better search functionality in the function I wrote ( `containsSymbolNode`) rather than using the old `lastSymbolNode` search used previously which relied on polynomial structure.

`'g *( x ) = (x-4)^(2) - 3` is a pretty crazy structure and proved to be too complex for that old method.